### ISSUE TITLE
Two different task with same task-id

### ISSUE THREAD

**Issue by swistak** [HabitRPG/habitica#7558]
General Info
- UUID: a847788d-5f96-4bf3-94c5-7e1ac7b4cb0c
- Browser: Firefox (newest)
- OS: Windows 7
### Description

Two separate tasks have been given the same task id: task-79f3230a-1e9c-4507-ae36-519871e3049e
This results in me not beeing able to complete the second task because all the calls always refer to the first one.
These task have not been modified recently so I don't know why did they get the same id suddenly. Not to mention that these id's should be unique. You need to review your UUID generation scheme.

PS. Funny enough you can work around this by moving the second task ahead of first one, but then the second one doesn't work. So you must switch it back and so on.

To summarize: unique fields are not unique, please fix that (preferably by setting a constraint on the field in database).

**Comment by Alys** [comment]
Unique IDs are enforced by our database and it is impossible to create a task that contains the same ID as an existing task. When an attempt to do that is made, an error is returned (code 11000, message "E11000 duplicate key error index...").

In addition, I've searched through the ID numbers of your tasks and found only one task with the ID 79f3230a-1e9c-4507-ae36-519871e3049e

I suspect your browser has corrupted some of the data it has stored. Please log out of Habitica and log back in.

If the problem persists, please give us more information what you're seeing, including console errors, and especially tell us what makes it look as if two tasks have the same ID number.
